rust/appveyor.yml

```diff
@@ -0,0 +1,16 @@
+clone_depth: 1

```

Comment:
No need for this file here? Or maybe add it in the main repository CI setup?

Reply:
The main repository doesn't have one at the moment. Nor does the other rz-pipe modules.
So, I think this can be added when a singular, global .yml is made.